requirements.txt

```diff
@@ -1,6 +1,7 @@
 click==6.2
 PTable==0.9.2
 netaddr==0.7.18
+nsot==1.1.1
```

Comment:
That's the goal since I'm using `nsot.util.slugify()` here:
https://github.com/dropbox/pynsot/pull/129/files#diff-0d16436d27089e20e56334a599c1b1e4R10

It _is_ kind of annoying that we pull in the entirety of `nsot`, but using `nsot.util.slugify()` means that if that slugify method is updated in the API, we don't have to worry about the client being incorrect.

Thinking more about this, I'm kind of on the fence about it since I don't see us updating `slugify()` very often. I can yank this dep and re-implement that function within pynsot if you'd like.